### ISSUE TITLE
Request: pause between reading different types of information

### ISSUE THREAD

**Issue by TechHorseG** [nvaccess/nvda#10318]
Feature request for the option to have a pause between the reading out of different kinds of information relating to an item (the name, the menu key, its place in the list etc). 

I sometimes have to listen twice as the information runs into eachother. So for example, read "Anne ... a" rather than "Anna", and "pic 3 ... 2 of 7" rather than "pic3 2of7"

**Comment by josephsl** [comment]
Hi, note that sometimes, this is speech synthesizer dependent, but I think we have the necessary foundations to bring this to reality. CC @LeonardDer, @JCSTeh, @DerekRiemer

**Comment by Qchristensen** [comment]
Quick comment that we have had requests for this.  I expect if we do change it that we will have people who preferred it how it is now, so perhaps an option for how much of a pause between elements from none upwards.

**Comment by TechHorseG** [comment]
@Qchristensen absolutely 100% please make any pause delays optional!

On the occasions I have had to go back to using Narrator temporarily, I have found myself getting quite irritated by having to wait for the pauses. For example in FileExplorer Narrator will read out:

Date modified pause... 01/02/2022 pause... 12:00
 
Quite irritating when you wish to quickly check the modification times of several files!

I was glad to see the back of the pauses when switching back to NVDA.

As the person who made this request I have come to realise that I would greatly resent mandatory new pauses.

The ideal solution for me would be a custom gesture to turn the pauses on and off for those times they are actually needed. But failing that, please, please make any new pause delays optional!

**Comment by Qchristensen** [comment]
I can see it being something which could be set from no pause upwards, and ideally also able to be saved in configuration profiles, that way you could have a pause in one program but not another.

**Comment by XLTechie** [comment]
Might it also be likely that some users will prefer earcons instead of pauses?

With that thought in mind, I would suggest the ability to turn on configurable 
earcons for various item types (menu shortcut keys, list items, whatever as 
discussed above), and to have one of the earcons available be a pause.
Or "short pause", "medium pause", "long pause".
Or some such thing (personally that's too many pause options, but some people 
will inevitably hate whatever singular pause length is chosen, so...)

I guess I am proposing an earcon panel in settings, and should do that on 
whatever thread is discussing earcons.

Anyway, it would seem reasonable to me, at least on the surface, to consider 
integrating this with whatever earcon work is being done/considered, and 
looking at pauses as just another earcon.

**Comment by josephsl** [comment]
Hi, that’s perhaps one of the most significant reasons for using, or requesting, sound schemes. We have several add-ons that can sort of do what Luke is proposing – the obvious question for at least one add-on is who will update it to make it more modern. Thanks.

**Comment by fettahpinar** [comment]
Hello.
I would also like the pause setting to be added.
especially when using acapela tts sounds, these sounds pause a lot.

**Comment by XLTechie** [comment]
> especially when using acapela tts sounds, these sounds pause a lot.

I'm not really sure if I understand what you mean there, @fettahpinar. If it's 
convenient/easy, can you make a recording of what you are talking about and post 
a link?

**Comment by Adriani90** [comment]
I think @fettahpinar expects a configurable pause between speak messages and controle types / states or between sentences. This issue explicitely requests an option for a longer pause, but I also understand that some synthesizers have already long pauses implemented, as stated in above comments.

**Comment by XLTechie** [comment]
Yes well, the One Core pauses are intolerable, so I am familiar with the 
problem. But this issue is about adding pauses, not subtracting them (which is 
the synth's responsibility, I don't think we can do that).

But @fettahpinar said that the "TTS sounds" pause a lot, and I wasn't sure if 
that was just another word for speech, or if something else was involved.

I do not know much about Acapela, except that I don't like their voice samples.

**Comment by jackf723** [comment]
I see this indirectly got added in 2024.1RC, best served with a synth other than ESpeak if ESpeak is at a fast rate. Menu options and their shortcut keys now have a pause in between them. So while we're at it,
1. Can this behavior be brought into list items and the list index as well?
2. The original issue of configurable pause length still stands, but Ideally it would be great to allow the option of having a different voice reading things such as menu shortcuts, list indexes, etc.
